src/features/import/hooks/useImportedFile.ts

```diff
@@ -0,0 +1,42 @@
+import { addFiles } from '../store';
+import { parseExcelFile } from '../utils/parseFile';
+import { useAppDispatch } from 'core/hooks';
+import { useState } from 'react';
+import { ImportedFile, parseCSVFile } from '../utils/parseFile';
+
+export default function useFileImport() {
```

Comment:
Name differs from name of file. I think the name of the function is best, so I would rename the file.

Reply:
This is not resolved in #1612